integration-libs/cdc/core/store/actions/cdc-user-token.action.ts

```diff
@@ -23,16 +23,32 @@ interface LoadUserTokenPayload {
   baseSite: string;
 }
 
-interface LoadUserTokenFailurePayload {
-  error: ErrorActionType;
-  initialActionPayload: LoadUserTokenPayload;
-}
-
```

Comment:
[LOAD]

Removed in favor of using `LoadUserTokenPayload` with `... & { error: <specific_type>`